core/Enclave/key_operation.cpp

```diff
@@ -738,7 +738,7 @@ sgx_status_t ehsm_rsa_decrypt(const ehsm_keyblob_t *cmk,
 
     if (plaintext->datalen == 0)
```

Comment:
commit msg
"
Fix the memory leak issue

free the memory after use:
1. sm2_crypto_test()
2. ehsm_rsa_decrypt()

test: passed all the unit test
Signed-off-by: wanghaiheng <haihengx.wang@intel.com>
"